trade-finance-manager-api/src/v1/emails/amendments/automatic-approval-email-variables.js

```diff
@@ -1,15 +1,23 @@
+const { DEAL_TYPE } = require('../../../constants/deals');
+
 const automaticAmendmentEmailVariables = (amendmentVariables) => {
   const { amendment, dealSnapshot, user } = amendmentVariables;
   const { ukefFacilityId } = amendment;
-  const { exporter, ukefDealId, bankInternalRefName } = dealSnapshot;
+  const { exporter, ukefDealId, bankInternalRefName, dealType, details } = dealSnapshot;
   const { firstname, surname } = user;
 
+  let ukefDealIdFromDeal = ukefDealId;
```

Comment:
can we use something like this:
`const ukefDealId = dealSnapshot.ukefDealId ?? dealSnapshot.details.ukefDealId` - same thing applies to the other files
This means, if the value exists and is not `null` or `undefined`, then get the ukefDealId. otherwise, search for a value inside the details object

Reply:
done